src/ec/packages/ec-component-breadcrumb-core/ecl-breadcrumb-core.html.twig

```diff
@@ -21,11 +21,14 @@
         ...
       ]
     - "_compliance" (boolean) (default: false) Activates debug
+    - "inner_check" (boolean) (default: false) Inline compliance report
```

Comment:
I am not convinced of the name of this parameter. why did you put undescore to the **_compliance_** parameter but not this one? Isn't it better to name it **_compliance_inner_check_**?

Reply:
Would it be nice to give a little more explanation on this parameter? Or update documentation as you describe the parameter in your PR description?

Reply:
The thing is that inner_check is basically an internal parameter because it's just the internal logic that sets it for the included templates.  The `_compliance_` one is the only relevant parameter from the user input data, `inner_check` is just offering a compact report if it's possible and it's mainly us deciding when to apply it, is not meant to be provided by the user.
I will add some documentation for us, the developers, then, about the name i don't know, i think i just wanted something "compact", but i understand that like this is not "connected" to the functionality, semantically speaking. I can rename it like you suggest.